Shootin

Fishervjacob opened the pull request:
added shoot component
DESTROYED GRABAGE!!!!

Commit:
Adding shooting stuff, NOTHING GETS SHOT YET!!
	modified:   src/main/java/com/bitdecay/ludum/dare/actors/player/Player.java
	deleted:    src/main/java/com/bitdecay/ludum/dare/components/GamepadControlComponent.java
	modified:   src/main/java/com/bitdecay/ludum/dare/components/KeyboardControlComponent.java
	new file:   src/main/java/com/bitdecay/ludum/dare/components/ShootComponent.java
	deleted:    src/main/java/com/bitdecay/ludum/dare/control/ControllerScreenObject.java
	modified:   src/main/java/com/bitdecay/ludum/dare/control/InputAction.java
	modified:   src/main/java/com/bitdecay/ludum/dare/screens/SetupScreen.java

Commit:
Merge branch 'master' of github.com:bitDecayGames/LudumDare36 into shootin
Mergin

jacque006 commented on src/main/java/com/bitdecay/ludum/dare/actors/player/Player.java:
```diff
@@ -39,9 +40,10 @@ public Player() {
 
-        append(size).append(pos).append(phys).append(health).append(jetpack).append(anim).append(keyboard);
+        append(size).append(pos).append(phys).append(health).append(jetpack).append(anim).append(keyboard).append(shoot);
+        phys.getBody().controller = new PlayerInputController(keyboard);
```
Why dis moved?

jacque006 commented on src/main/java/com/bitdecay/ludum/dare/components/GamepadControlComponent.java:
```diff
@@ -1,105 +0,0 @@
-package com.bitdecay.ludum.dare.components;
```
Leave this in, we might want this later.

jacque006 commented on src/main/java/com/bitdecay/ludum/dare/components/ShootComponent.java:
```diff
@@ -0,0 +1,23 @@
+            //TODO shoot a thing
+        }
+    }
+}
```
This may be better handled by a state rather than a separate component.

Commit:
Gamepad not deleted
fix weirdness
weeeeeirdneeeeessssss.....
	modified:   src/main/java/com/bitdecay/ludum/dare/actors/player/Player.java
	new file:   src/main/java/com/bitdecay/ludum/dare/components/GamepadControlComponent.java
	modified:   src/main/java/com/bitdecay/ludum/dare/control/InputAction.java

jacque006 commented:
+1
